cypress/e2e/common/common.js

```diff
@@ -1066,17 +1144,55 @@ Given("I create a tenure {string} {string}", (startOfTenureDate, isResponsible)
 })
 
 After(() => {
-    const filename = "cypress/fixtures/recordsToDelete.json";
-    cy.readFile(filename)
-      .then(data => {
+  const filename = "cypress/fixtures/recordsToDelete.json";
+  cy.readFile(filename)
+    .then(data => {
+      return new Cypress.Promise((resolve) => {
+        Promise.all(data.map(record => dynamoDb.deleteRecord(record)))
+          .then(() => {
+            resolve()
+          })
+      }).then(() => {
+        cy.writeFile(filename, []);
+        cy.log("Test database records cleared!")
+      })
+    });
+})
+
+beforeEach(() => {
```

Comment:
beforeEach and afterEach (line 1181) have been added so that test data added to DynamoDb is cleared before and after each test.